### DISCUSSION TITLE
Request for sass theme file and reporting a broken link for primeblocks

### DISCUSSION THREAD

**Discussion by giosueDelgado** [primefaces/community#2743] [PrimeNG Blocks]
Hello,

We purchased the full version of PrimeBlocks a few months ago.

Could you please provide us with the SASS theme file for PrimeBlocks?

Additionally, I would like to report a broken link on this page: https://blocks.primeng.org/#/documentation.

FYI @hnone

**Comment by scsete** [comment]
https://www.primefaces.org/primeblocks-ng doesn't work either... dead with 404
<img width="1346" alt="image" src="https://github.com/user-attachments/assets/c6d447d7-eef2-4e6a-bcd5-2860ee31d8c0">
